src/module.ts

```diff
@@ -297,8 +293,25 @@ export default defineNuxtModule<ModuleOptions>({
       level: 999,
       global: true
     })
-    // Register user global components
 
+    addTemplate({
+      filename: 'types/content.d.ts',
+      getContents: () => [
+        'declare module \'#content/server\' {',
+        `  import { QueryBuilder, ParsedContent, QueryBuilderParams } from "${relative(nuxt.options.buildDir, resolve('./runtime/types'))}";`,
+        '  const serverQueryContent: <T = ParsedContent>() => QueryBuilder<T>;',
```

Comment:
I think the types are wrong, see https://github.com/nuxt/content/blob/feat/nitro-alias/src/runtime/server/storage.ts#L90

Reply:
Yes, I must mistakenly pushed my tests, before the `serverQueryContent` PR.